echopype/convert/set_groups_ek60.py

```diff
@@ -656,78 +621,63 @@ def set_beam(self) -> List[xr.Dataset]:
                 ),
             }
 
-            if not self.parsed2zarr_obj.temp_zarr_dir:
-                var_dict["backscatter_r"] = (
-                    ["ping_time", "range_sample"],
-                    self.parser_obj.ping_data_dict["power"][ch],
-                    {
-                        "long_name": self._varattrs["beam_var_default"]["backscatter_r"][
-                            "long_name"
-                        ],
-                        "units": "dB",
-                    },
-                )
+            var_dict["backscatter_r"] = (
+                ["ping_time", "range_sample"],
+                self.parser_obj.ping_data_dict["power"][ch],
+                {
+                    "long_name": self._varattrs["beam_var_default"]["backscatter_r"]["long_name"],
+                    "units": "dB",
+                },
+            )
```

Comment:
This is a trivial comment: to move the setting of this variable into the above dict with all the other variables